docs/book/src/deploy/rama-cli.md

```diff
@@ -31,7 +31,7 @@ Options:
 The easiest way to install `rama` is by using `cargo`:
 
 ```sh
-cargo install rama-cli@0.2.0-alpha.8
+cargo install rama-cli@0.2.0-alpha.9
```

Comment:
revert changes in this file please, the install instructions are always of the last released version, as the current main version is always the next (not yet published) one